lib/rules/meta-viewport.json

```
@@ -3,7 +3,14 @@
   "selector": "meta[name=\"viewport\"]",
   "matches": "is-initiator-matches",
   "excludeHidden": false,
-  "tags": ["cat.sensory-and-visual-cues", "wcag2aa", "wcag144", "ACT"],
+  "tags": [
+    "cat.sensory-and-visual-cues",
+    "wcag2aa",
+    "wcag144",
+    "ACT",
+    "TTv5",
+    "TT18.a"
+  ],
```

There's nothing specific about either testing in mobile, or looking for viewports. I don't think this is covered by Trusted Tester.

```suggestion
  "tags": ["cat.sensory-and-visual-cues", "wcag2aa", "wcag144", "ACT"],
```